src/rules/noVarRequiresRule.ts

```diff
@@ -52,11 +66,19 @@ class NoVarRequiresWalker extends Lint.ScopeAwareRuleWalker<{}> {
     public visitCallExpression(node: ts.CallExpression) {
         const expression = node.expression;
 
+        const options = this.getOptions() as any[];
+        const patternConfig = options[0] as { "ignore-module": string } | undefined;
+        const ignorePattern = patternConfig === undefined ? undefined : new RegExp(patternConfig[OPTION_IGNORE_MODULE]);
```

Comment:
consider moving this block of code right before the value is actually used to avoid unnecessary work